src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlConnectionFactory.cs

```diff
@@ -206,44 +206,18 @@ override protected DbConnectionPoolGroupOptions CreateConnectionPoolGroupOptions
             return poolingOptions;
         }
 
-        // SxS (VSDD 545786): metadata files are opened from <.NetRuntimeFolder>\CONFIG\<metadatafilename.xml>
-        // this operation is safe in SxS because the file is opened in read-only mode and each NDP runtime accesses its own copy of the metadata
-        // under the runtime folder.
-        [ResourceExposure(ResourceScope.None)]
-        [ResourceConsumption(ResourceScope.Machine, ResourceScope.Machine)]
         override protected DbMetaDataFactory CreateMetaDataFactory(DbConnectionInternal internalConnection, out bool cacheMetaDataFactory)
         {
             Debug.Assert(internalConnection != null, "internalConnection may not be null.");
-            cacheMetaDataFactory = false;
 
-            if (internalConnection is SqlInternalConnectionSmi)
-            {
-                throw SQL.NotAvailableOnContextConnection();
-            }
-
-            NameValueCollection settings = (NameValueCollection)PrivilegedConfigurationManager.GetSection("Microsoft.Data.SqlClient");
-            Stream XMLStream = null;
-            if (settings != null)
-            {
-                string[] values = settings.GetValues(_metaDataXml);
-                if (values != null)
-                {
-                    XMLStream = ADP.GetXmlStreamFromValues(values, _metaDataXml);
```

Comment:
Seems like we're dropping this way of populating xmlStream.
Could you elaborate more if it could have any impact?

Reply:
The way I read the previous code was that this was attempting to read the Microsoft.Data.SqlClient.SqlMetaData.xml stuff first from the system's machine.config. Then when that failed, it would read from Microsoft.Data.SqlClient.SqlMetaData.xml. Since SqlClient is no longer shipped within the .NET Framework itself and that content will never be there, there is no need to try and read it from there.